docs/developer-docs/latest/plugins/email.md

```diff
@@ -6,219 +6,105 @@ canonicalUrl: https://docs.strapi.io/developer-docs/latest/plugins/email.html
+// This code example can be used in a controller or a service
+// path: ./src/api/{api name}/controllers/{api name}.js or ./src/api/{api name}/services/{api name}.js 
+
+  await strapi.plugins['email'].services.email.send({
+    to: 'valid email address',
+    from: 'your verified email address', //e.g. single sender verification in SendGrid
+    cc: 'valid email address',
+    bcc: 'valid email address',
+    replyTo: 'valid email address',
+    subject: 'The Strapi Email plugin worked successfully',
+    text: 'Hello world!',
+    html: 'Hello world!',
+  }),
+```
 
-When you send an email, you will most likely want to build it from a template you wrote.
-The email plugin provides the service `sendTemplatedEmail` that compile the email and then sends it. The function have the following params:
+### Using the `sendTemplatedEmail()` function
 
-| param           | description                                                                                                              | type   | default |
-| --------------- | ------------------------------------------------------------------------------------------------------------------------ | ------ | ------- |
-| `emailOptions`  | Object that contains email options (`to`, `from`, `replyTo`, `cc`, `bcc`) except `subject`, `text` and `html`            | object | `{}`    |
-| `emailTemplate` | Object that contains `subject`, `text` and `html` as [lodash string templates](https://lodash.com/docs/4.17.15#template) | object | `{}`    |
-| `data`          | Object that contains the data used to compile the templates                                                              | object | `{}`    |
+The `sendTemplatedEmail()` is used to compose emails from a template. The function compiles the email from the available properties and then sends the email. To use the `sendTemplatedEmail()` function, define the `emailTemplate` object and add the function to a controller or service. The function calls the `emailTemplate` object, and can optionally call the `emailOptions` and `data` objects.
 
-**Example**
+| Parameter       | Description                                                                                                                                | Type     | Default |
+|-----------------|--------------------------------------------------------------------------------------------------------------------------------------------|----------|---------|
+| `emailOptions`  | Contains email addressing properties: `to`, `from`, `replyTo`, `cc`, and `bcc`                                                             | `object` | { }      |
+| `emailTemplate` | Contains email content properties: `subject`, `text`, and `html` using [Lodash string templates](https://lodash.com/docs/4.17.15#template) | `object` | { }      |
+| `data`          | Contains the data used to compile the templates                                                                                            | `object` | { }      |
```

Comment:
`data` seems optional (I can't find it in the code example, maybe I misunderstood something 🤔) so I think you can mention it in the table (e.g. "`data`<br/>(_optional_)" 
The intro paragraph implies that emailOptions is optional as well, but is it? I'm trying to figure out how the email is sent if neither `to` or `from` parameters are defined (just an open question) 🤔

Reply:
Added "optional" to the table. 

I think the answer is because you can also hard code the email address information. Here the source code, which as I understand, only throws an error is `emailTemplate` is not supplied. 

```const sendTemplatedEmail = (emailOptions = {}, emailTemplate = {}, data = {}) => {
  const attributes = ['subject', 'text', 'html'];
  const missingAttributes = _.difference(attributes, Object.keys(emailTemplate));
  if (missingAttributes.length > 0) {
    throw new Error(
      `Following attributes are missing from your email template : ${missingAttributes.join(', ')}`
    );
  }```